samples/spring-eventsourced-shopping-cart/src/test/java/com/example/shoppingcart/domain/ShoppingCartTest.java

```diff
@@ -18,24 +18,31 @@ public class ShoppingCartTest {
   @Test
   public void testAddLineItem() {
 
-    EventSourcedTestKit<ShoppingCart, ShoppingCartEntity> testKit = EventSourcedTestKit.of(ShoppingCartEntity::new); // <1>
+    var testKit = EventSourcedTestKit.of(ShoppingCartEntity::new); // <1>
```

Comment:
note that the callouts is out of view and requires horizontal scrolling. By using `var` we reduce the line length quite a bit and they become visible.